WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -504,9 +504,20 @@ - (void)setRestorableSelectedIndexPath:(NSIndexPath *)restorableSelectedIndexPat
 {
     _restorableSelectedIndexPath = restorableSelectedIndexPath;
```

Comment:
I have a somewhat related concern with `_restorableSelectedIndexPath` being modified in its `- (NSIndexPath *)restorableSelectedIndexPath` `getter`. It does not look like that will affect the crash though.